index.d.ts

```diff
@@ -5,41 +5,44 @@
 
 export declare function testConnection(uri: string): Promise<string>
 export const enum CqlTypes {
-  Ascii = 0,
-  Boolean = 1,
-  Blob = 2,
-  Counter = 3,
-  Decimal = 4,
-  Date = 5,
-  Double = 6,
-  Duration = 7,
-  Empty = 8,
-  Float = 9,
-  Int = 10,
-  BigInt = 11,
-  Text = 12,
-  Timestamp = 13,
-  Inet = 14,
-  List = 15,
-  Map = 16,
-  Set = 17,
-  UserDefinedType = 18,
+  Ascii = 1,
+  BigInt = 2,
+  Boolean = 3,
+  Blob = 4,
+  Counter = 5,
+  Decimal = 6,
+  Double = 7,
+  Float = 8,
+  Int = 9,
+  Timestamp = 11,
+  Uuid = 12,
+  Text = 13,
+  Varint = 14,
+  Timeuuid = 15,
+  Inet = 16,
+  Date = 17,
+  Time = 18,
   SmallInt = 19,
   TinyInt = 20,
-  Time = 21,
-  Timeuuid = 22,
-  Tuple = 23,
-  Uuid = 24,
-  Varint = 25
+  Duration = 21,
+  List = 32,
+  Map = 33,
+  Set = 34,
+  UserDefinedType = 48,
+  Tuple = 49,
+  Empty = 50
 }
```

Comment:
Those values are based on: 
https://github.com/scylladb/scylla-rust-driver/blob/4649acbd9f63c0fcd0fc2e52d66fcbd253a3110f/scylla-cql/src/frame/response/result.rs#L629-L652
Although the current state of this PR doesn't use any particular mappings to the integer in this enum, it may become useful later as the datastax driver uses the same values for corresponding types: https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/6d1c8208910f942982b58d8642079d7deb485e4d/lib/types/index.d.ts#L22-L50

Pleas note, that values in this file you mentioned are auto-generated from https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/1a90383c901f3da8ea9110aef55cb7157febbd86/src/result.rs#L31-L59